monitor/README.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: pattern
+title: Monitor
+folder: monitor
+permalink: /patterns/monitor/
+categories: Concurrency
+tags:
+ - Performance
+---
+
+## Intent
+Monitor pattern is used to create thread-safe object . In the concurrent application to prevent conflict between threads .
+## Explanation
+
+In plain words
+
+> Monitor pattern is used when we have critical section or race condition on shared resource in your application so we have to use mutual exclusion to solve this problem ! one of the ways to achieve mutual exclusion is Monitor pattern that invented by Tony Hoare.
+
+Wikipedia says
+
+> In concurrent programming (also known as parallel programming), a monitor is a synchronization construct that allows threads to have both mutual exclusion and the ability to wait (block) for a certain condition to become false. Monitors also have a mechanism for signaling other threads that their condition has been met. 
+
```

Comment:
Add here brief programmatic example how the pattern is implemented referencing the sample code. See the other patterns' `README.md` for examples.

Reply:
done !